test/unit/reporters/html/html.test.js

```diff
@@ -13,25 +13,25 @@ const path = require('path');
 const lib = require('lib/reporters/html/lib');
 const view = require('lib/reporters/html/view');
 
-describe('HTML Reporter', () => {
-    const sandbox = sinon.sandbox.create();
-    let emitter;
-
-    function mkStubResult_(options) {
-        return _.defaultsDeep(options, {
-            state: {name: 'name-default'},
-            browserId: 'browserId-default',
-            suite: {
-                path: ['suite/path-default'],
-                metaInfo: {sessionId: 'sessionId-default'}
-            },
-            saveDiffTo: sandbox.stub(),
-            currentPath: 'current/path-default',
-            referencePath: 'reference/path-default',
-            equal: false
-        });
-    }
+const sandbox = sinon.sandbox.create();
+let emitter;
+
+function mkStubResult_(options) {
+    return _.defaultsDeep(options, {
+        state: {name: 'name-default'},
+        browserId: 'browserId-default',
+        suite: {
+            path: ['suite/path-default'],
+            metaInfo: {sessionId: 'sessionId-default'}
+        },
+        saveDiffTo: sandbox.stub(),
+        currentPath: 'current/path-default',
+        referencePath: 'reference/path-default',
+        equal: false
+    });
+}
```

Comment:
Because i used these functions in both describes - but no longer an issue, since I updated the test file. Viz. next comment.